corehq/apps/cloudcare/templates/formplayer/query_view.html

```diff
@@ -88,6 +88,7 @@ <h2 tabindex="0"><%- title %></h2>
     </fieldset>
 
     <% } else if (input == "address") { %>
+    <div role="region" aria-live="polite" class="sr-only" id="<%- id %>-sr"></div>
```

Comment:
Can we add some comment/documentation here?

"region, used by screen reader is populated via javascript"

and then in query.js
"this populates the region in query view for screen readers"

or something better. We need to make it really explicit why we have these "region" divs in the html and the event tracking to populate these, is hard to even find, since even the id is dynamically generated.